### ISSUE TITLE
scrollBack is broken

### ISSUE THREAD

**Issue by javierarce** [Vizzuality/NEEMO#27]
I'm not really sure why, but I've tracked down the bug to this commit: 8e955fcc3924719a6a10b01f0eb00e451f8384c5

**Comment by andrewxhill** [comment]
Yup, sorry bout that. It actually only works to comment out backBuffer. What happens, is that when I backBuffer, if the region hasn't been loaded before it .prepend()s the new .image in the div. But it gets the alignment out of what. We can focus  on this when you have time.

**Comment by andrewxhill** [comment]
Arce, I've given this one back to you. If you comment in line 262 of Slideshow.js it will allow endless scrollback. But, you'll see that after the second one it starts shifting the entire screen right.

**Comment by andrewxhill** [comment]
There are a few issues related to this. If you look at my just pushed code. In Slideshow line 212 I commented out a bit of code. What I was going to do there is start removing regions that you are far past. But if I now remove divs from the beginning of the image set, it messes up our offset and the track starts shifting left. We should work together to handle some of this.

**Comment by javierarce** [comment]
Ok, I'll take a look this weekend.

You are referring to these lines https://github.com/Vizzuality/NEEMO/blob/master/app/public/js/Slideshow.js#L220-238, right?

**Comment by andrewxhill** [comment]
yup, that's the spot, and this

https://github.com/Vizzuality/NEEMO/blob/master/app/public/js/Slideshow.js#L307
